lib/rules/no-computed-in-data.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * @fileoverview Ensure computed properties are not used in the data()
+ * @author IWANABETHATGUY
+ */
+'use strict'
+
+// ------------------------------------------------------------------------------
+// Rule Definition
+// ------------------------------------------------------------------------------
+const utils = require('../utils')
+module.exports = {
+  meta: {
+    type: 'problem',
+    docs: {
+      description: 'require computed properties are not used in the data()',
+      categories: ['strongly-recommended'],
```

Comment:
Could you change the categories to be set to undefined

Reply:
![image](https://user-images.githubusercontent.com/17974631/78466503-4a7e0900-7734-11ea-945b-66e7dd17506e.png)
if i set the categories to be undefined, the eslint will report this error

Reply:
i change the categories same as `vue/no-side-effects-in-computed-properties`, is that ok?

Reply:
i misunderstand your meaning before, i set it to undefined already😂